tutor-multi-chart/templates/echo.yaml

```diff
@@ -45,7 +45,7 @@ kind: Ingress
 metadata:
   name: cluster-echo-test
 spec:
-  ingressClassName: tutor-multi-traefik
+  ingressClassName: {{ ternary (index .Values "ingress-nginx" "controller" "ingressClass")  "tutor-multi-traefik" (index .Values "ingress-nginx" "enabled") }}
```

Comment:
We can remove `tutor-multi-traefik` from here - I don't think it's needed anymore.